rln/src/circuit.rs

```diff
@@ -8,34 +8,35 @@ use num_bigint::BigUint;
 use serde_json::Value;
 use std::convert::TryFrom;
 use std::fs::File;
+use std::io::Read;
 use std::io::{Cursor, Error, ErrorKind, Result, Write};
 use std::option::Option;
 use std::path::Path;
 use std::str::FromStr;
 
+use once_cell::sync::{Lazy, OnceCell};
+use std::sync::Mutex;
+use wasmer::{Module, Store};
+
 const ZKEY_FILENAME: &str = "rln_final.zkey";
 const VK_FILENAME: &str = "verifying_key.json";
-const R1CS_FILENAME: &str = "rln.r1cs";
```

Comment:
Is the r1cs file not useful for debugging? https://github.com/iden3/r1csfile/blob/master/doc/r1cs_bin_format.md

Reply:
I expect most of the (circuit) debugging to be done in circom, not in zerokit. Here we just plug a verified/tested circuit compiled in circom and use it to generate proofs. If I get your concerns correctly, the debugging phase should happen before the compiled circuit is embedded in zerokit. Of course you might find some unexpected behaviors while using zerokit, but then it would most probably be a bad circuit implementation, hence you're back to circom. Does this address your concern?

Reply:
hmm, not sure

I think you are correct most debugging will happen at circom level, but I also see the need for debugging generally as well as for perf etc in Zerokit

but let's cross that bridge when we get there and ignore for now